collector/test/HostHeuristicsTest.cpp

```diff
@@ -65,36 +61,17 @@ class MockCollectorConfig : public CollectorConfig {
   }
 };
 
-TEST(HostHeuristicsTest, TestS390XRHEL84) {
-  MockS390xHeuristics s390xHeuristics;
-  MockHostInfoHeuristics host;
-  KernelVersion version = KernelVersion("4.18.0-305.88.1.el8_4.s390x", "", "s390x");
-  MockCollectorConfig config;
-  HostConfig hconfig;
-
-  config.SetCollectionMethod(CollectionMethod::EBPF);
-  hconfig.SetCollectionMethod(CollectionMethod::EBPF);
-  EXPECT_CALL(host, GetKernelVersion()).WillOnce(Return(version));
-
-  s390xHeuristics.Process(host, config, &hconfig);
-
-  EXPECT_EQ(hconfig.GetCollectionMethod(), CollectionMethod::CORE_BPF);
-}
-
-TEST(HostHeuristicsTest, TestARM64Heuristic) {
-  MockARM64Heuristics heuristic;
+TEST(HostHeuristicsTest, TestCPUHeuristic) {
```

Comment:
I'm a bit confused as to why is this test being added as part of this PR, it doesn't feel like it's related to the logging changes. Just for the sake of cleanliness, maybe we could consider moving it to a separate PR? Even if it ends up being just 20 lines of changes.

Reply:
This is a fun one. If I remove all the tests from this file then the compilation of the unit tests fails at link time because there's certain functions omitted from the suite, so the "fix" is to just delete the file. 

But I thought it's a bit silly to delete if we have some stuff here for testing HostHeuristics anyway, so I thought I could just add one to keep the test suite alive.

Reply:
I mean, I think I would just delete the file, but to each their own.

Reply:
but to delete it and then re-add it in a subsequent PR feels like more effort, and this heuristic doesn't have a test so I thought it'd be worth fixing now. The others still need tests I think, but since they all call CLOG(FATAL), they're a bit trickier and should probably be done in a follow up